runtime/kusama/src/lib.rs

```diff
@@ -303,6 +304,10 @@ impl pallet_balances::Config for Runtime {
 	type MaxReserves = MaxReserves;
 	type ReserveIdentifier = [u8; 8];
 	type WeightInfo = weights::pallet_balances::WeightInfo<Runtime>;
+	type FreezeIdentifier = ();
+	type MaxFreezes = ();
```

Comment:
Why is the `MaxFreezes` set to 0?

Reply:
Well, nothing it using it and the ID type `()` is definitely not compatible with the final aggregate enum.

Reply:
But `MaxFreezes` is not an ID type? It is a `Get<u32>`, think `ConstU32<0>` could be less confusing.